src/components/molecules/SfProperty/SfProperty.html

```diff
@@ -0,0 +1,6 @@
+<div class="sf-property">
+	<slot>
+        <span class="sf-property__property">{{ property }}:</span>
+        <router-link class="sf-property__value" to="/">{{ value }}</router-link>
```

Comment:
imho, would be good to provide a named slot here (and bind the value to slot), maybe `<slot name="value" :value="value">`, wdyt?
and `router-link` it's a default for vue-router, it's case specific so I think we shoudn't use by default, my suggestion is a normal link with `href="javascript:;"`

Reply:
I think you may also emit an event on click, returning the property object (with name and value)